From 065a37f376b5d87a5812dc639c3bcfba0ef2e919 Mon Sep 17 00:00:00 2001
From: Adam Seaton <aseaton@google.com>
Date: Fri, 12 Oct 2018 13:41:15 -0700
Subject: [PATCH 4/4] [DO NOT MERGE] Update Security String to 2018-12-01 for
 December Release bug:117667960 (cherry picked from commit
 d6edd2123ce5f3ef100657c0a416d25bc4de81b7)

Change-Id: I92db5581f988978254df2ad2da7307c21e699d3e
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index d2df00e22..e8edf6a09 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -231,7 +231,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-      PLATFORM_SECURITY_PATCH := 2018-11-05
+      PLATFORM_SECURITY_PATCH := 2018-12-01
 endif
 
 ifndef PLATFORM_SECURITY_PATCH_TIMESTAMP
-- 
2.17.1

